.prettierignore

```diff
@@ -0,0 +1,19 @@
+# exclude everything by default
+*.*
+# exclude these files
+Dockerfile
+# exclude these directories
+/assets/
+/bin/
+/config/
```

Comment:
Are you sure we shouldn't format `bin` or `config`?

Reply:
Not now, but soon 😈